app/_src/guides/otel-metrics.md

```diff
@@ -95,7 +95,7 @@ With configuration in place we can install OpenTelemetry collector:
 
 ```shell
 helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
-helm install --namespace mesh-observability opentelemetry-collector open-telemetry/opentelemetry-collector -f values-otel.yaml
+helm install --namespace mesh-observability opentelemetry-collector open-telemetry/opentelemetry-collector -f values-otel.yaml --set image.repository=otel/opentelemetry-collector-contrib
```

Comment:
Hmm I think this needs an explanation, why are we setting this image?

Reply:
ok, I gonna supplement it.